_episodes/01-quickstart.md

```diff
@@ -29,8 +31,8 @@ keypoints:
   configuration file"
 - "Use `esmvaltool run <recipe>.yml` to run a recipe"
 ---
->
-> This quickstart guide is compatible with ESMValTool v2.6.0.
+
+
  {: .callout}
```

Comment:
I think that would solved the failed test
```suggestion

```

Reply:
Fixed this and the version numbers.